tests/tests/tier0/monitor-node-disconnect/python/monitor.py

```diff
@@ -0,0 +1,30 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
Is this the correct license?

Reply:
Currently, yes. It'll change later, I think.

Reply:
It will be changed after finishing of the relicensing process, we still need to wait on the remaining acks